src/components/bodyImage.tsx

```diff
@@ -26,7 +26,8 @@ const imgStyles = (width: number, height: number): SerializedStyles => css`
     background: ${neutral[97]};
     display: block;
     width: 100%;
- 
+    object-fit: cover;
```

Comment:
In theory the CSS is sizing this box to be the exact dimensions of the image, but doesn't hurt to be safe 👍

Reply:
Maybe it's related to the side paddings. Removing `width: 100%` also maintains the aspect-ratio, but with it they aren't the exact dimensions